src/utils/constants.ts

```diff
@@ -14,3 +14,6 @@ export const cooldownTime = 60000;
 
 // cooldown for ever box highlight
 export const detectCooldown = 5000;
+
+//cool down for every fetched user location
+export const FetchLocation_CoolDown = 5000;
```

Comment:
please follow the naming convention with the other variables and use camel case.